test/users/ferguson/histogram/reductionex_class.bad

```diff
@@ -1,6 +1,30 @@
-Note: This source location is a guess.
-reductionex_class.chpl:34: error: Illegal use of identifier 'NBUCKETS' from enclosing type
-Note: This source location is a guess.
-reductionex_class.chpl:35: error: Illegal use of identifier 'NBUCKETS' from enclosing type
-Note: This source location is a guess.
+reductionex_class.chpl:18: error: illegal use of identifier 'max' from enclosing type
+reductionex_class.chpl:16: note: the identifier is used within class 'myhisto', declared here
+reductionex_class.chpl:11: note: however, the identifier refers to a field of an enclosing class 'Test', declared here
+reductionex_class.chpl:13: note: field declared here
+reductionex_class.chpl:18: error: illegal use of identifier 'min' from enclosing type
+reductionex_class.chpl:16: note: the identifier is used within class 'myhisto', declared here
+reductionex_class.chpl:11: note: however, the identifier refers to a field of an enclosing class 'Test', declared here
+reductionex_class.chpl:14: note: field declared here
+reductionex_class.chpl:18: error: illegal use of identifier 'NBUCKETS' from enclosing type
+reductionex_class.chpl:16: note: the identifier is used within class 'myhisto', declared here
+reductionex_class.chpl:11: note: however, the identifier refers to a field of an enclosing class 'Test', declared here
+reductionex_class.chpl:12: note: field declared here
+reductionex_class.chpl:19: error: illegal use of identifier 'NBUCKETS' from enclosing type
+reductionex_class.chpl:16: note: the identifier is used within class 'myhisto', declared here
+reductionex_class.chpl:11: note: however, the identifier refers to a field of an enclosing class 'Test', declared here
+reductionex_class.chpl:12: note: field declared here
+reductionex_class.chpl:21: In method 'identity':
+reductionex_class.chpl:22: error: illegal use of identifier 'NBUCKETS' from enclosing type
+reductionex_class.chpl:16: note: the identifier is used within class 'myhisto', declared here
+reductionex_class.chpl:11: note: however, the identifier refers to a field of an enclosing class 'Test', declared here
+reductionex_class.chpl:12: note: field declared here
+reductionex_class.chpl:34: In method 'accumulate':
+reductionex_class.chpl:34: error: illegal use of identifier 'NBUCKETS' from enclosing type
+reductionex_class.chpl:16: note: the identifier is used within class 'myhisto', declared here
+reductionex_class.chpl:11: note: however, the identifier refers to a field of an enclosing class 'Test', declared here
+reductionex_class.chpl:12: note: field declared here
+reductionex_class.chpl:35: error: illegal use of identifier 'NBUCKETS' from enclosing type
+reductionex_class.chpl:16: note: the identifier is used within class 'myhisto', declared here
+reductionex_class.chpl:11: note: however, the identifier refers to a field of an enclosing class 'Test', declared here
+reductionex_class.chpl:12: note: field declared here
```

Comment:
This doesn't need to be addressed in this PR, but I feel like a newline between chunks of multiline errors would be useful.